app.js

```diff
@@ -1,5 +1,45 @@
-// import functions and grab DOM elements
+import {
+    getRandomThrow,
+    didUserWin
+} from './get-random-throw.js'
 
-// initialize state
+const shootButton = document.getElementById('shoot-button');
+const winSpan = document.getElementById('win-span');
+const lossSpan = document.getElementById('loss-span');
+const drawSpan = document.getElementById('draw-span');
+const throwResults = document.getElementById('throw-results');
 
-// set event listeners to update state and DOM
\ No newline at end of file
+let wins = 0;
+let losses = 0;
+let draws = 0;
+
+shootButton.addEventListener('click', () => {
+    const compThrow = Math.floor(Math.random() * 3) + 1;
+    const rockPaperScissors = getRandomThrow(compThrow);
+    const selectedRadioButton = document.querySelector('input[type="radio"]:checked');
+    const userPlay = selectedRadioButton.value;
```

Comment:
i'm getting an error in your deployed site that says "cannot read property value of null" here. time to follow the bouncing ball the "selectedRadioButton" ! :)